src/v2/guide/computed.md

```diff
@@ -77,6 +77,8 @@ You can open the console and play with the example vm yourself. The value of `vm
 
 You can data-bind to computed properties in templates just like a normal property. Vue is aware that `vm.reversedMessage` depends on `vm.message`, so it will update any bindings that depend on `vm.reversedMessage` when `vm.message` changes. And the best part is that we've created this dependency relationship declaratively: the computed getter function has no side effects, which makes it easier to test and understand.
 
```

Comment:
"cannot be to" >> "cannot be used to",

Also, can you add 2 quick code examples, one not working with v-for, and another working with a method?

For tips, use '<p class="tip">Note that...</p>'